docs/source/json-api/lf-value-specification.rst

```diff
@@ -17,12 +20,19 @@ The suggested defaults for both of these flags is false. If the intended
 recipient is written in JavaScript, however, note that the JavaScript data
 model will decode these as numbers, discarding data in some cases;
 encode-as-String avoids this, as mentioned with respect to ``JSON.parse``
-below. For that reason, the HTTP JSON API Service uses ``true`` for both flags.
+below. **For that reason, the HTTP JSON API Service uses ``true`` for both flags.**
+
+Type-directed Parsing
+*********************
 
 Note that throughout the document the decoding is type-directed. In
 other words, the same JSON value can correspond to many Daml-LF values,
```

Comment:
We could also say here, "and vice versa: a Daml-LF value can correspond to two or more JSON encodings"